fish_command_timer.fish

```diff
@@ -123,7 +123,7 @@ else
   set -e fish_command_timer_enabled
 end
 
-set fish_command_timer_start_time
+set fish_command_timer_start_time (fish_command_timer_get_ts)
```

Comment:
That is very strange - I don't encounter that error at all. What version of fish are you running (`fish --version`)?

Logically, since the preexec function (which sets this variable) will execute before each command, by the time the postexec function runs, it's guaranteed to be already set. So there's no reason to set it here. Even if it's set to a value, that value would be useless (it's the timestamp when this file was included, not the timestamp before a command line is executed). So we might as well set it to "dummy" if it's for compatibility.